R/grid_helpers.R

```diff
@@ -541,3 +541,12 @@ set_workflow_recipe <- function(workflow, recipe) {
   workflow$pre$actions$recipe$recipe <- recipe
   workflow
 }
+
+
+do_future_quietly <- function (code) {
+  old <- options()
+  options(doFuture.rng.onMisuse = "ignore")
```

Comment:
```suggestion
  old <- options(doFuture.rng.onMisuse = "ignore")
```

I'm fairly certain `options()` always returns the "old" options, even when you set a new one, since using it in an `on.exit()` is so common

Reply:
I didn't that this was in `rlang`!